src/routes/+error.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { page } from '$app/stores';
+</script>
+
+<!-- if the status code is "404" and the error message is "not found" -->
+{#if $page.status === 404 && $page.error.message === "Not Found"}
```

Comment:
404 status is always "Not Found" so no need for the 
```svelte
{#if $page.status === 404 && $page.error.message === "Not Found"}
```
could just be `{#if $page.status === 404}`

Reply:
Will do the quick Fix.
Thanks Boss